src/XAD/UnaryOperators.hpp

```diff
@@ -497,4 +497,4 @@ XAD_INLINE float copysign(float x, const Expression<Scalar, Derived>& y)
 #undef XAD_MAKE_UNARY_FUNC
 #undef XAD_MAKE_FPCLASSIFY_FUNC
 #undef XAD_MAKE_FPCLASSIFY_FUNC_RET
-}  // namespace xad
+}  // namespace xad
```

Comment:
What kind of change was this? Please make sure there is a newline at the end of the file.

Reply:
If this is a coding convention for XAD, I believe you should include it in [CONTRIBUTING](https://github.com/auto-differentiation/xad/blob/main/CONTRIBUTING.md). If this is specific to this file, it was an oversight from my part and will be fixed.

Reply:
We use clang-format mainly for coding conventions like this. We prefer not to over-specify the conventions as it can be off-putting to contributors. However, we try to keep the diffs small in PRs - only the necessary changes - to avoid unrelated conflicts. 

However, I just saw that clang-format [added a setting for this](https://releases.llvm.org/16.0.0/tools/clang/docs/ClangFormatStyleOptions.html#insertnewlineateof) - perhaps you can add this flag and reformat the files.